src/juju/api.ts

```diff
@@ -113,14 +111,12 @@ function determineLoginParams(
   credentials: Credential | null | undefined,
   authMethod?: AuthMethod,
 ) {
-  let loginParams: Credentials = {};
   if (credentials && authMethod === AuthMethod.LOCAL) {
-    loginParams = {
+    return {
       username: credentials.user,
       password: credentials.password,
     };
   }
-  return loginParams;
```

Comment:
Maybe we could include `return {}` here in order to be consistent with the return type of `Credentials`?